templates/miq-template.yaml

```diff
@@ -446,6 +524,8 @@ objects:
             value: "${POSTGRESQL_MAX_CONNECTIONS}"
           - name: POSTGRESQL_SHARED_BUFFERS
             value: "${POSTGRESQL_SHARED_BUFFERS}"
+          - name: POSTGRESQL_CONFIG_DIR
```

Comment:
@carbonin They do not need to rebuild the image to change this, kubernetes will allow the configmap to be placed anywhere on the filesystem, it has special privileges. The baked default directory is handy when things go wrong with the configmap, when is either missing or deleted, or also older builds such as euwe which will begin using newer PG images that support conf.d but their templates are old and do not supply the configmap yet.